ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-netobserv-perf-tests.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  ocp-qe-perfscale-ci:
+    name: ocp-qe-perfscale-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: Dockerfile.prow
+  inputs:
+    ocp-qe-perfscale-ci:
+      as:
+      - registry.ci.openshift.org/ci/ocp-qe-perfscale-ci:latest
+  to: netobserv-perfscale
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: test-deploy
+  run_if_changed: scripts/
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
```

Comment:
Is that possible to use `aws-qe` profile? `aws-perfscale-qe` is for perfscale team usage.

Reply:
I responded on slack thread, I think it makes sense to use `aws-perfscale-qe` cluster_profile since these configs are for perfscale actually so that it cloud usage/costs could be attributed to perfscale tests,  let me know if you think differently.